### PR TITLE
Ms.connections

### PR DESCRIPTION
**Pull request by mariano54**

### CODE REVIEW

File: src/daemon/client.py [hunk 1]
```diff
@@ -22,7 +22,9 @@ def format_request(self, command, data=None):
         return request
 
     async def start(self):
-        self.websocket = await websockets.connect(self._uri)
+        self.websocket = await websockets.connect(
+            self._uri, max_size=(1024 * 1024 * 30)
```
Comment (hoffmang9) [inline src/daemon/client.py:26]: Did you mean 10MB or 30MB?

File: src/util/initial-config.yaml [hunk 2]
```diff
@@ -133,7 +133,7 @@ full_node:
   # How often to connect to introducer if we need to learn more peers
   introducer_connect_interval: 500
   # Continue trying to connect to more peers until this number of connections
-  target_peer_count: 10
+  target_peer_count: 15
```
Comment (hoffmang9) [inline src/util/initial-config.yaml:136]: Just curious about your thinking here?

File: electron-react/src/electron-wait-react.js [hunk 3]
```diff
@@ -1,7 +1,8 @@
 const net = require("net");
 const port = process.env.PORT ? process.env.PORT - 100 : 3000;
+const config = require("./util/config");
 
-process.env.ELECTRON_START_URL = `http://localhost:${port}`;
+process.env.ELECTRON_START_URL = `http://${config.self_hostname}:${port}`;
```
Comment (hoffmang9) [inline electron-react/src/electron-wait-react.js:5]: This should be of interest to @freddiecoleman so I want to make sure he's had a look at this too.

**Review by hoffmang9 — APPROVED** [review]
With only that inconsistency around 30MB v 10MB and my query - this looks good to me

File: src/daemon/client.py (hunk 1 above)
Comment (mariano54) [reply src/daemon/client.py:26]: 30 MB. Pyl is already seeing sizes of 3MB in some messages (he has thousands of plots).

File: src/util/initial-config.yaml (hunk 2 above)
Comment (mariano54) [reply src/util/initial-config.yaml:136]: There are more and more people joining testnet, and our nodes don't have enough space. I just want to ensure that there are enough upnp nodes out there for anyone to join the network. Also more connections =more security, and more load testing.

File: electron-react/src/electron-wait-react.js (hunk 3 above)
Comment (mariano54) [reply electron-react/src/electron-wait-react.js:5]: Yeah this was motivated by his docker use case, now there is only one place to change your IP (config.yaml)

**Comment by mariano54** [comment]
Thanks, please don't merge yet, I would like to add fee input field for standard wallet and coloured coin wallet.

File: electron-react/src/electron-wait-react.js (hunk 3 above)
Comment (freddiecoleman) [reply electron-react/src/electron-wait-react.js:5]: Just noticed this.

I run ran dev branch in a container and got it to work after changing the `config.yaml`.

Thank you.

**Pull request merged**